src/main/java/teammates/common/util/Const.java

```diff
@@ -1403,6 +1403,9 @@ private Const() {
         public static final String STUDENT_COURSE_JOIN_SUCCESSFUL = "You have been successfully added to the course %s.";
         
         /** Template String. Parameters:  Course ID */
+        public static final String DELETED_STUDENT_ATTEMPTING_TO_JOIN =
+                "Unable to join course %s as you have been unenrolled by an instructor. Please contact your "
+                + "course instructor for assistance.";
```

Comment:
We may need to rephrase the message to avoid situation where a student email the instructor "How come you unenrolled me?" and the instructor replies "I did no such thing". In rare occasions, this situation can be caused by other issues such as eventual consistency, so it is better to phrase the message is a non-committal way.

How about this?

`Unable to join course %s as you are currently not in the student list of that course. Please contact your course instructor for assistance.`